WinImageBuilder.psm1

```diff
@@ -148,6 +152,7 @@ function Create-ImageVirtualDisk {
         [string]$DiskLayout
     )
 
+Write-Log "Create Virtual Disk Image"

```

Comment:
Make sure you have the correct indentation.

Moreover, do a mirror logging. This means logging the start of an action and the end of the action.
For example:

Write-Log "Creating virtual disk $VhdPath..."
<after time and executions>
Write-Log "Successfully created virtual disk $VhdPath."
or, if errors:
Write-Log "Failed to create virtual disk $VhdPath with error $error"

Reply:
Rewrite all the logging in this manner.